Package.swift

```diff
@@ -19,6 +19,7 @@ var products: [Product] = [
   .executable(name: "sourcekit-lsp", targets: ["sourcekit-lsp"]),
   .library(name: "_SourceKitLSP", targets: ["SourceKitLSP"]),
   .library(name: "LSPBindings", targets: ["LanguageServerProtocol", "LanguageServerProtocolJSONRPC"]),
+  .library(name: "InProcessLSPClient", targets: ["InProcessClient"]),
```

Comment:
Should we name the product the same as the target?

```suggestion
  .library(name: "InProcessClient", targets: ["InProcessClient"]),
```

Reply:
I'm 50-50 on that, and it's up to you (and others) who maintain the project to ultimately tell me what makes more sense.

I went with `InProcessLSPClient` to closer match the rest of the product names and be more descriptive to package consumers what is in that product

Reply:
Let’s go with `InProcessClient` to keep it consistent with the target name. Since you’re depending on the sourcekit-lsp package, the LSP is already implied.

Reply:
Updated!